scripts/gen-go-service-sdk.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+set -x
+
+# This script regenerates the Go SDK for a given HCP service ($1).
+
+# The steps are:
+# 1. Remove the original SDK files for the service if they exist.
+# 2. Fetch the latest public API specs for the service from the central spec repo.
+# 3. Run temporary transformations on those specs to prepare them for SDK generation.
+# 4. Iterate over each stage and version of the service specs and generate the corresponding SDK. 
+#    (Note: Currently only the 'preview' stage is supported)
+# 5. Remove temporary directories.
+
+SCRIPTS_DIR=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
We might be able to add something like `ROOT_DIR=$(dirname $SCRIPTS_DIR)`, and then use `$ROOT_DIR` throughout the script. The reason I bring it up is the `realpath` command is not available everywhere by default (it isn't on my Mac), so I think using it here could be something that unsuspecting devs could trip over.

Reply:
Actually I realized last night I misdiagnosed the issue, thinking go-swagger expected the absolute path. But it doesn't! It was actually all the [glob's fault](https://unix.stackexchange.com/questions/314804/how-to-make-bash-glob-a-string-variable) 🙈

Reply:
Oh nice, even better!